README.md

```diff
@@ -55,11 +55,10 @@ Your guests will configure themselves.
 
 Be sure to start and run the environment with `make start` before you
 continue with these steps. You must have working vagrant, virtualbox, and
-ansible.
+ansible. If you are behind a proxy server, then use only http proxy.
```

Comment:
@erikh: Ansible task will fail with https proxy.
TASK: [ceph-common | install the ceph stable repository key] ******************
failed: [mon1] => {"failed": true}
msg: Unsupported proxy scheme: https. Currently ansible only supports HTTP proxies.
failed: [mon0] => {"failed": true}
msg: Unsupported proxy scheme: https. Currently ansible only supports HTTP proxies.
failed: [mon2] => {"failed": true}
msg: Unsupported proxy scheme: https. Currently ansible only supports HTTP proxies.

FATAL: all hosts have already failed -- aborting